### DISCUSSION TITLE
Welcome to mad-libs Discussions!

### DISCUSSION THREAD

**Discussion by Campbell-E-Soup** [Campbell-E-Soup/mad-libs#1] [Announcements]
Welcome!

**Comment by HarvestMint** [comment]
[Teams Project Chat](https://teams.microsoft.com/l/team/19%3AuPwgzmXcAP8EEUirP5YK4y88-WTvc7Ofs_rr4FzDJCI1%40thread.tacv2/conversations?groupId=3223072f-b7ac-4d3f-9143-1e0db3f31d03&tenantId=4e4d2ea9-72a7-4448-b440-72defa6963b6)
